crates/cheatnet/src/forking/state.rs

```diff
@@ -37,14 +38,16 @@ pub struct ForkStateReader {
     client: JsonRpcClient<HttpTransport>,
     block_number: BlockNumber,
     runtime: Runtime,
-    cache: ForkCache,
+    cache: RefCell<ForkCache>,
```

Comment:
Why?

Reply:
`StateReader` trait has been changed to take in `&self` instead of `&mut self`

Reply:
Oh and we need to modify the cache, okay, got it